samples/tab-deeplink-all-features/csharp/DeeplinkAllFeatures/wwwroot/css/style.css

```diff
@@ -0,0 +1,65 @@
+pre {
+    overflow-x: scroll;
+}
+
+.callDeeplinkButton {
+    font-family: "Segoe UI", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Web";
+    font-weight: 700;
+    background-color: #444791;
+    cursor: pointer;
+    color: white;
+    width: auto;
+    border-radius: 3px;
+    border: 1px;
+    margin-top: 1rem;
+    padding: 10px 10px;
+    text-align: center;
+    text-decoration: none;
+}
+
+.callDeeplinkButton:hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.inputContainer {
+    margin-top: 10px;
+}
+
+.container {
+    font-family: "Segoe UI", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Web";
+    font-size: 14px;
+}
+
+.card {
+    box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%);
+    width: 27rem;
+    min-height: 18rem;
+    overflow: hidden;
+    margin-left: 2rem;
+    margin-top: 0.5rem;
+}
+
+.card-container-div {
+    padding-bottom: 2rem;
+    display: flex;
+}
+
+.card-header {
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-top: 1rem;
+    padding-top: 0.5rem;
+    font-size: 16px !important;
```

Comment:
I don't think we are using all of these .css files.

Reply:
Removed which are not used